sdk/ml/azure-ai-ml/azure/ai/ml/_utils/azure_resource_utils.py

```diff
@@ -0,0 +1,67 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+from typing import List, Dict, Optional
+import azure.mgmt.resourcegraph as arg
+from azure.mgmt.resource import SubscriptionClient, ResourceManagementClient
+from azure.core.credentials import TokenCredential
+
+
+def get_resources_from_subscriptions(strQuery: str, credential: TokenCredential,
```

Comment:
This is a public method without any docstring.

Reply:
This is not meant to be a public method. It's under the `_utils` namespace, so it should not be public.